app/posts/hpv-self-sampling/2025/03/2025-03-31-hpv-self-sampling-work-so-far.md

```diff
@@ -0,0 +1,224 @@
+---
+title: HPV Self-Sampling - Design work so far
+description: This is a recap of what our design team has done in Q3/Q4 2024-25.
+date: 2025-03-31
+tags:
+  - cervical screening
+  - hpv self sampling
+  - research
+  - workshops
+---
+
+
```

Comment:
You could add an summary paragraph here, before the sub-headings, to give people a quick overview of what the post is about?